msteams-platform/includes/tabs/content-security-policy-headers.md

```diff
@@ -1,5 +1,14 @@
-|Microsoft 365 app host| frame-ancestor permission|
+> [!WARNING]
+> Microsoft's cloud services, including web versions of Teams (*teams.microsoft.com*), Outlook (*outlook.com*), and Microsoft 365 (*microsoft365.com*) domains are migrating to the new *cloud.microsoft* domain by June 2024. We recommend you to perform the following steps before June 2024 to ensure your app renders on the Teams web client:
+>
+> 1. Update TeamsJS SDK to v.2.19.0 or higher. For more information about the latest release of TeamsJS SDK, see [Microsoft Teams JavaScript client library](https://www.npmjs.com/package/@microsoft/teams-js).
+>
+> 2. Update your [`X-Frame-Options`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options) or [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) headers in your Teams app to allow your app to access the ***teams.cloud.microsoft*** domain.
+>
+> 3. If your Teams app extends across Outlook and Microsoft 365, ensure you allow your app to access ***teams.cloud.microsoft***, ***outlook.cloud.microsoft***, and ***m365.cloud.microsoft*** domains.
```

Comment:
Could we combine these steps, since the supporting links apply to both Teams/other hosts? Also visually seems less daunting of a customer ask. (Also, `x-frame-options` are N/A for this specific CTA) Suggestion:

> 2. Update your [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) headers in your app to allow access to the ***teams.cloud.microsoft*** domain. If your Teams app extends across Outlook and Microsoft 365, ensure you also allow ***outlook.cloud.microsoft***, and ***m365.cloud.microsoft*** domains.

Reply:
Done